addons/binding/org.openhab.binding.avmfritz/ESH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="avmfritz"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Channel definitions of all FRITZ! devices -->
+	<channel-type id="mode">
+		<item-type>String</item-type>
+		<label>Mode</label>
+		<description>States the mode of the device (MANUAL/AUTOMATIC).
+		</description>
+		<state pattern="%s" readOnly="true">
+			<options>
+				<option value="MANUAL">Manual</option>
+				<option value="AUTOMATIC">Automatic</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="locked" advanced="true">
+		<item-type>Contact</item-type>
+		<label>Device locked (external)</label>
+		<description>Device is locked for switching over external sources.
+		</description>
+		<category>Contact</category>
+		<state pattern="%s" readOnly="true" />
+	</channel-type>
+
+	<!-- Channel definitions of specific FRITZ! devices -->
+	<channel-type id="temperature">
+		<item-type>Number</item-type>
+		<label>Actual Temperature</label>
+		<description>Current measured temperature (in °C).</description>
+		<category>Temperature</category>
+		<state pattern="%.1f °C" readOnly="true" />
+
+		<config-description-ref uri="channel-type:avmfritz:temperature-channel" />
```

Comment:
No, it is not supported. It only used it for testing translations of channel-type configs.